app/__init__.py

```diff
@@ -11,13 +12,20 @@
 from flask_migrate import Migrate
 from flask_jwt_extended import JWTManager
 from decouple import config as db_config
+from app import celeryapp
+
 
 db = SQLAlchemy()
 ma = Marshmallow()
 migrate = Migrate()
 login_manager = LoginManager()
 jwt = JWTManager()
 
+
+DEBUG = db_config("DEBUG", default=True)
+get_config_mode = "Debug" if DEBUG else "Production"
```

Comment:
shouldn't  this  be
`get_config_mode = "Development" if DEBUG else "Production"` 
or rather
`get_config_mode = "Dev" if DEBUG else "Production"`
?

Reply:
I have changed it from `"Debug"` to `"Development"`